src/components/homepage/homepage.js

```diff
@@ -0,0 +1,73 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import axios from 'axios';
+import { v4 as uuidv4 } from 'uuid';
+import {
+  Container, Row, InputGroup, FormControl, Button,
+} from 'react-bootstrap';
+import { getList, filterList } from '../../redux/homepage/homepage';
+import Items from './items';
+import Header from '../header';
+
+const HomePage = () => {
+  const items = useSelector((state) => state.homeReducer);
+  const dispatch = useDispatch();
+
+  const CountriesList = () => {
+    axios.get('https://api.covid19tracking.narrativa.com/api/2021-08-01')
+      .then((response) => {
+        const list = [];
+        let [countries] = [...Object.values(response.data.dates)];
+        countries = countries.countries;
+        Object.entries(countries).forEach((element) => {
+          if (element[1].regions.length !== 0) {
+            list.push({
+              country: element[0],
+              confirmed_cases: element[1].today_confirmed,
+            });
+          }
+        });
+        dispatch(getList(list));
+      });
+  };
+
+  useEffect(() => {
+    CountriesList();
+  }, []);
+
+  return (
+    <>
+      <Header title="countries" backButton />
+      <Container className="countries-description">
+        <Row>
+          <h3>Confirmed cases of covid19 until 2021-08-01</h3>
+        </Row>
+        <Row>
+          <InputGroup className="mb-3">
+            <FormControl
+              placeholder="Filter by country"
+            />
+            <Button variant="outline-light" onClick={(e) => dispatch(filterList(e.target.previousElementSibling.value))}>
```

Comment:
- [ ] OPTIONAL: The sibling element target here which is the search input element is supposed to be a state. You should use the useState hook to initialize the filter state and update it when the input value is changed. Therefore filterList function can take filter as an argument. Now the search form is not working at all. I wanna make this change optional because it is possible filter by clicking on the category cards and I think that is enough for the minimum requirement. 

![Screenshot from 2021-09-19 01-49-06](https://user-images.githubusercontent.com/64894927/133906902-15c6a34b-983e-4582-b442-488dd6a41f0a.png)